code/frameworks/nextjs/src/config/webpack.ts

```diff
@@ -54,3 +56,23 @@ const setupRuntimeConfig = (baseConfig: WebpackConfig, nextConfig: NextConfig):
 
   baseConfig.plugins?.push(new DefinePlugin(definePluginConfig));
 };
+
+export const applyNextConfigWebpackConfig = (
+  baseConfig: WebpackConfig,
+  nextConfig: NextConfig
+): void => {
+  if (typeof nextConfig.webpack === 'function') {
+    nextConfig.webpack(baseConfig, {
+      dev: process.env.NODE_ENV === 'development',
+      isServer: false,
+      buildId: nextConfig.buildId || '',
+      dir: '.',
+      config: nextConfig as NextConfigComplete,
+      defaultLoaders: {
+        babel: {},
+      },
+      totalPages: 0,
+      webpack: baseConfig,
```

Comment:
I am not sure about this. I can't find it on the documentation page: https://nextjs.org/docs/app/api-reference/next-config-js/webpack